benches/benches/bevy_transform/transform_hierarchy.rs

```diff
@@ -0,0 +1,655 @@
+        },
+    ),
+    (
+        "deep_tree",
+        Cfg {
+            test_case: TestCase::NonUniformTree {
+                depth: 25,
+                branch_width: 2,
+            },
+            update_filter: UpdateFilter {
+                probability: 0.5,
+                min_depth: 0,
+                max_depth: u32::MAX,
+            },
+        },
+    ),
+    (
+        "chain",
+        Cfg {
+            test_case: TestCase::Tree {
+                depth: 2500,
+                branch_width: 1,
+            },
+            update_filter: UpdateFilter {
+                probability: 0.5,
+                min_depth: 0,
+                max_depth: u32::MAX,
+            },
+        },
+    ),
+    (
+        "update_leaves",
+        Cfg {
+            test_case: TestCase::Tree {
+                depth: 18,
+                branch_width: 2,
+            },
+            update_filter: UpdateFilter {
+                probability: 0.5,
+                min_depth: 17,
+                max_depth: u32::MAX,
+            },
+        },
+    ),
+    (
+        "update_shallow",
+        Cfg {
+            test_case: TestCase::Tree {
+                depth: 18,
+                branch_width: 2,
+            },
+            update_filter: UpdateFilter {
+                probability: 0.5,
+                min_depth: 0,
+                max_depth: 8,
+            },
+        },
+    ),
+    (
+        "humanoids_active",
+        Cfg {
+            test_case: TestCase::Humanoids {
+                active: 4000,
+                inactive: 0,
+            },
+            update_filter: UpdateFilter {
+                probability: 1.0,
+                min_depth: 0,
+                max_depth: u32::MAX,
+            },
+        },
+    ),
+    (
+        "humanoids_inactive",
+        Cfg {
+            test_case: TestCase::Humanoids {
+                active: 10,
+                inactive: 3990,
+            },
+            update_filter: UpdateFilter {
+                probability: 1.0,
+                min_depth: 0,
+                max_depth: u32::MAX,
+            },
+        },
+    ),
+    (
+        "humanoids_mixed",
+        Cfg {
+            test_case: TestCase::Humanoids {
+                active: 2000,
+                inactive: 2000,
+            },
+            update_filter: UpdateFilter {
+                probability: 1.0,
+                min_depth: 0,
+                max_depth: u32::MAX,
+            },
+        },
+    ),
+];
+
+// Random seed for tree creation
+// (kept constant to make benchmark results comparable)
+const SEED: u64 = 0x94eb0d25004f5f17;
+
+criterion_group!(
+    transform_hierarchy_benches,
+    all_benches,
+);
+
+fn build_app(cfg: &Cfg, enable_update: bool) -> App {
+    let mut app = App::new();
+
+    app.add_plugins(TransformPlugin);
+
+    if enable_update {
+        app
+            .add_plugins(TimePlugin)
+            // Updating transforms *must* be done before `CoreSet::PostUpdate`
```

Comment:
There is no such thing as a `CoreSet` anymore. Where does this come from?

Reply:
This is part of the code I simply copied over from the stress test, so it's probably an oversight from that change: https://github.com/bevyengine/bevy/blob/505b9a58bd8768c7d31446ee9bb23639cd127cc5/examples/stress_tests/transform_hierarchy.rs#L188-L189
I removed the comment in the benchmark (but not in the example).